lib/TabTracker.js

```diff
@@ -13,7 +13,9 @@ function TabTracker(trackableURLs, clientID) {
   this._clientID = clientID;
   this._trackableURLs = trackableURLs;
   this.onOpen = this.onOpen.bind(this);
+  this.onLoadComplete = this.onLoadComplete.bind(this);
   tabs.on("open", this.onOpen);
+  Services.obs.addObserver(this.onLoadComplete, "performance-log-complete");
```

Comment:
nit: please use a `const` for consistency; an upstream version of the file uses `const`s instead of strings:
https://github.com/mozilla/activity-streams/blob/59806985ab1138fe9fbc83f352f3ab4d53cc7eb8/lib/TabTracker.js